docs/index.asciidoc

```diff
@@ -523,6 +523,13 @@ should reference only data or client nodes in Elasticsearch.
 Any special characters present in the URLs here MUST be URL escaped! This means
 `#` should be put in as `%23` for instance.
 
+When using Logstash 7.9+, this setting supports defining an environment variable
+`hosts => '${ES_HOSTS:}' with multiple {logstash-ref}/plugin-concepts.html[whitespace-delimited URIs], e.g.:
+
+```
+ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
+```
```

Comment:
Suggested text/example:

>{ls} 7.9 and later supports defining an environment variable `hosts =>
'${ES_HOSTS}' with multiple whitespace delimited URIs. 
This plugin automatically expands `hosts` strings that contain multiple
white-space delimited URs into separate entries. 

>**Example**
>```
>ES_HOSTS="es1.example.com es2.example.com:9201 es3.example.com:9201"
>```

Reply:
"This plugin automatically expands hosts strings that contain multiple
white-space delimited URs into separate entries." - I feel like this is very developer centric and less easy to understand.

Reply:
What about something like 

> This setting supports defining an environment variable containing one or more hostnames, each separated by whitespace 

Followed by an example of the environment variable definition(s), and how to use it/them in the configuration. With a note that prior to `7.9`, the environment variable would only support a single hostname?

Reply:
++